api/src/main/java/jakarta/servlet/jsp/el/ImplicitObjectELResolver.java

```diff
@@ -285,7 +285,11 @@ public boolean isReadOnly(ELContext context, Object base, Object property) {
      *                <code>null</code> return value.
      * @return An <code>Iterator</code> containing one <code>FeatureDescriptor</code> object for each implicit object,
      *         or <code>null</code> if <code>base</code> is not <code>null</code>.
+     *         
+     * @deprecated This method is deprecated as of EL 5.0 and will be removed in EL 6.0 (Jakarta EE 11). Therefore it
+     *             will be removed here in JSP 4.0.
      */
+    @Deprecated(forRemoval = true, since = "JSP 3.1")
```

Comment:
A big :+1: from me on this one.

Reply:
Tx for the quick review. We have relatively little time left to complete the work for the next release so timely reviews are a huge help.